### DISCUSSION TITLE
Using signal outside of reactive tracking context in 0.7 (fine in 0.6)

### DISCUSSION THREAD

**Discussion by nlfiedler** [leptos-rs/leptos#3365] [General]
This code, or one of several instances like it, is causing an error, maybe? It's the only code using `ColorMode` so it would have to be here.

```
<i class=move || {
  if mode.get() == ColorMode::Dark {
    "fa-solid fa-moon"
   } else {
    "fa-solid fa-sun"
  }
}
aria-hidden="true"></i>
```

The error message in the console:

```
At /Users/nfiedler/.cargo/registry/src/index.crates.io-6f17d22bba15001f/reactive_graph-0.1.0/src/wrappers.rs:477:35, you access a reactive_graph::signal::arc_read::ArcReadSignal<leptos_use::use_color_mode::ColorMode> (defined at /Users/nfiedler/.cargo/registry/src/index.crates.io-6f17d22bba15001f/reactive_graph-0.1.0/src/signal/rw.rs:180:40) outside a reactive tracking context. This might mean your app is not responding to changes in signal values in the way you expect.
```

The entire file is in this gist: https://gist.github.com/nlfiedler/2af5814091fa3087f1200314147f6b7b

Am I accessing the signal incorrectly? This was working fine in 0.6.

All of the source in question, if you're interested, is here (https://github.com/nlfiedler/tanuki/tree/upgrade/src/preso/leptos) on a branch. It's about 4,700 lines, but the `home.rs` has been stripped to nearly nothing because I'm trying to debug hydration errors.

Thank you for reading.

**Comment by gbj** [comment]
Is something broken, or do you think this is a false positive?

If it's a false positive, then reducing it to an actual minimal reproduction would be helpful. There are probably some missing `#[track_caller]` annotations in either `reactive_graph` or `leptos_use` that can help get the line numbers better, but it is way easier to work with a smaller example.

(By "minimal reproduction," what I mean is keep deleting lines until the warning no longer appears, and until removing anything from the example causes the warning no longer to appear. The hard thing working with "here's the whole file" examples like this is that they pull in other parts of your crate, other dependencies, etc. and it's easy to spend 15 minutes trying to set up a reproduction by guessing what those are only to find that it doesn't reproduce given the guesses)

**Comment by nlfiedler** [comment]
I'm sorry, I gave up after several hours over the past week. I could not figure out what was going wrong, even when comparing my code to the template. A minimal sample would work, but my app did not. Clearly something is wrong with my app, simple as that.
